base/logging.cc

```diff
@@ -1301,13 +1301,21 @@ ScopedLoggingSettings::ScopedLoggingSettings()
 ScopedLoggingSettings::~ScopedLoggingSettings() {
   // Re-initialize logging via the normal path. This will clean up old file
   // name and handle state, including re-initializing the VLOG internal state.
+#if defined(STARBOARD)
```

Comment:
Investigate CHECK macros and why they don't expand correctly. Fine to follow up with this